views/index.handlebars

```diff
@@ -51,7 +51,9 @@
         Uploading
       </div>
       <div class="upload-window">
+        <!-- htmllint id-no-dup="false" -->
         <div id="upload-img"><img src="/resources/upload.svg" alt="Upload" /></div>
+        <!-- htmllint tag-bans="$previous" -->
```

Comment:
This should "fix" #82 by having htmllint ignore that rule for that block. Although refactoring to use classes or redoing the UI will also fix it for reals.